src/cli/utils/exchange.go

```diff
@@ -1,8 +1,7 @@
 package utils
 
 import (
-	"errors"
-
+	"github.com/alcionai/clues"
```

Comment:
Almost had this one, though the rest of the imports could use some help
```suggestion
	"github.com/alcionai/clues"
	
```